.github/renovate.json5

```
@@ -217,6 +217,13 @@
         "quay.io/cilium/hubble"
       ],
     },
+    {
+      // Group golangci-lint updates.
```

Comment explains the what, but not the why - let's add some rationale here too, maybe (probably needs reformat)
```suggestion
      // Group golangci-lint updates to override schedule of GHA file udpates. Without this, golangci-lint updates are not in sync for GHA files and other usages.
```

---

@bimmlerd thanks for the valuable input!